src/TimePicker/TimePickerDialog.js

```diff
@@ -89,6 +89,7 @@ class TimePickerDialog extends Component {
       },
       dialogContent: {
         width: 280,
+        userSelect: 'none',
```

Comment:
Same comment. That would make things more deterministic. https://github.com/callemall/material-ui/blob/master/src/TimePicker/Clock.js#L126.